src/video/gl_backend.c

```diff
@@ -17,11 +17,12 @@ enum texture_map {
 enum uniform_attr {
   UNIFORM_MVP = 0,
   UNIFORM_DIFFUSE = 1,
-  UNIFORM_NUM_UNIFORMS = 2,
+  UNIFORM_PUNCH_THROUGH_ALPHA = 2,
```

Comment:
Let's stick with `UNIFORM_PT_ALPHA_REF`. The goal being to make it easier to search the code for `pt_alpha_ref` to understand how this all works in the future.